ui/src/theme.ts

```diff
@@ -63,7 +63,7 @@ const customTheme = createTheme({
     },
     subtitle1: {
       fontSize: '1rem', // Equivalent to 16px (16 * 1)
-      fontWeight: 400,
+      fontWeight: 600,
```

Comment:
will this affect any other component? is subtitle1 being use as a variant for any other?

Reply:
I think we need to change the theme.ts. I noticed yy followed the design specs in the beginning for the typography. But the design does not follow those design specs :") so.....